LPS-205004 Remove original poshi test

sandrodw3 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 commented:
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2235. [See changes here.](https://github.com/sandrodw3/liferay-portal/compare/f36c2b9e3fea~3...f36c2b9e3fea) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 committed:
LPS-205004 Add util to generate random id

Commit:
LPS-205068 remove the first link and update the message key and message of the second link

Commit:
LPS-205068 update message key

Commit:
LPS-205068 create new language key

sandrodw3 committed:
LPS-205004 Update package-lock.json

sandrodw3 committed:
LPS-205004 Remove original poshi test

Commit:
LPS-206312 Map the entire values array by the sortable field name for both indexType: keyword and text

Commit:
LPS-205068 Remove message key

Commit:
LPS-206312 Add unit test. Adjust _createDDMFormValues to accept more than one value.

Commit:
LPS-205068 update message key

izaera committed:
LPS-206193 Implement fixture

izaera committed:
LPS-206193 Disable unused vars SF rule

Since playwright relies heavily on the variables injected by the fixtures
to know whether to execute them, we cannot remove unused vars from test()
invocations since that would defeat the purpose of fixtures like the one
from this PR.

4lejandrito committed:
LPS-203508 Supporting both ce and dxp modes

izaera committed:
LPS-206513 Ignore playwright-report folder in playwright SF

sandrodw3 committed:
LPS-205004 Add utility to create a page

sandrodw3 committed:
LPS-205004 Add HeadlessSiteApiHelper with utilities to create and delete a site

sandrodw3 committed:
LPS-205004 Add utilities to get fragment and page definitions

sandrodw3 committed:
LPS-205004 Add PageEditorPage

sandrodw3 committed:
LPS-205004 Add test for checking configuration sections visibility

sandrodw3 committed:
LPS-205004 Move to types folder to follow pattern

sandrodw3 committed:
LPS-205004 Add content page and site related types

sandrodw3 committed:
LPS-205004 Sort

Commit:
LPS-205068 buildLang

marcosapmf committed:
LRAC-13481 use a thread-safe map implementation

brianchandotcom committed:
LPS-206312 SF
